### ISSUE TITLE
Add the HTML for BackApp

### ISSUE THREAD

**Issue by SgtGarro-zz** [SgtGarro/IHC#4]

**Comment by SgtGarro** [comment]
``<!DOCTYPE html>

<html data-wf-page="628ed7ff06cabf2dfe31b705" data-wf-site="628ed7fe06cabf316a31b704">
<head>
  <meta charset="utf-8">
  <title>Bienvenido a BackApp</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="css/normalize.css" rel="stylesheet" type="text/css">
  <link href="css/webflow.css" rel="stylesheet" type="text/css">
  <link href="css/el-garros-awesome-site.webflow.css" rel="stylesheet" type="text/css">
  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" type="text/javascript"></script>
  <script type="text/javascript">WebFont.load({  google: {    families: ["Inter:100,200,300,regular,500,600,700,800,900:cyrillic,cyrillic-ext,greek,greek-ext,latin,latin-ext,vietnamese"]  }});</script>
  
  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">
</head>
<body class="body">
  <div class="header"><img src="images/Logo.svg" loading="lazy" alt="" class="logo_backapp">
    <div class="nav">
      <a href="#" class="nav-link">Producto</a>
      <a href="#" class="nav-link">Planes</a>
      <a href="#" class="nav-link">Sobre nosotros</a>
      <a href="#" class="nav-link">Misión y Visión</a>
    </div>
  </div>
  <div class="container">
    <div class="presentation">
      <div class="title-pres">Mantente seguro gracias a BackApp</div>
      <div class="desc-pres">Software inteligente con el objetivo de brindarte el mejor apoyo contra la delincuencia y favorecer a la seguridad</div>
      <div class="buttons-pres">
        <a href="#" class="button-1-pres w-button">Conoce mas</a>
        <a href="#" class="button-2-pres w-button">Ver planes</a>
      </div><img src="images/Frame-3.svg" loading="lazy" alt="" class="image-pres">
    </div>
    <div class="about-backapp">
      <div class="title-characteristics">¿Cómo es nuestro producto?</div>
      <div class="characteristics">
        <div class="charac-desc"><img src="images/Group-11.svg" loading="lazy" alt="" class="image-1-char">
          <div class="title-char">Interfaz agradable y dinámica</div>
          <div class="text-char">Fácil de comprender para todo el publico general</div>
        </div>
        <div class="charac-desc"><img src="images/Group-10.svg" loading="lazy" alt="" class="image-2-char">
          <div class="title-char">Trabajo en segundo plano</div>
          <div class="text-char">El aplicativo realizara las acciones principales de manera automatica</div>
        </div>
        <div class="charac-desc"><img src="images/Group-9.svg" loading="lazy" alt="" class="image-3-char">
          <div class="title-char">Uso de potentes algoritmos</div>
          <div class="text-char">Menor riesgo de problemas o conflictos con el funcionamiento del software</div>
        </div>
      </div>
    </div>
    <div class="backapp-features">
      <div class="title-features">BackApp Features</div>
      <div class="desc-features">Conoce las principales funciones y/o características de nuestra aplicación para combatir contra la inseguridad en los usuarios</div>
      <div class="features">
        <div class="feature"><img src="images/Group-15.svg" loading="lazy" alt="" class="image-feature-1">
          <div class="block-background-feature">
            <div class="title-feature">Visualización de rutas seguras</div>
            <div class="text-feature">Brindar un mapa sobre las rutas menos peligrosas de los distritos</div>
          </div>
        </div>
        <div class="feature"><img src="images/Group-40.svg" loading="lazy" alt="" class="image-feature-2">
          <div class="block-background-feature">
            <div class="title-feature">Bloqueo de emergencia</div>
            <div class="text-feature">Bloqueo maestro del dispositivo en caso de robo</div>
          </div>
        </div>
        <div class="feature"><img src="images/Group-41.svg" loading="lazy" alt="" class="image-feature-3">
          <div class="block-background-feature">
            <div class="title-feature">Compartir tu estado</div>
            <div class="text-feature">Compartir ubicación y estado a los usuarios autorizados</div>
          </div>
        </div>
        <div class="feature"><img src="images/Group-14.svg" loading="lazy" alt="" class="image-feature-4">
          <div class="block-background-feature">
            <div class="title-feature">Notificación a las autoridades</div>
            <div class="text-feature">En caso de peligro, el algoritmo contactara con las autoridades</div>
          </div>
        </div>
        <div class="feature"><img src="images/Group-16.svg" loading="lazy" alt="" class="image-feature-5">
          <div class="block-background-feature">
            <div class="title-feature">Mejora continua del software</div>
            <div class="text-feature">Mejora en las actualizaciones gracias al feedback</div>
          </div>
        </div>
        <div class="feature"><img src="images/Group-17.svg" loading="lazy" alt="" class="image-feature-6">
          <div class="block-background-feature">
            <div class="title-feature">Sistema reconocedor de voz</div>
            <div class="text-feature">Identifica por el comportamiento del usuario el estado actual</div>
          </div>
        </div>
      </div>
    </div>
    <div class="backapp-benefits">
      <div class="content-benefits">
        <div class="title-benefits">¿Cuáles son los beneficios de usar BackApp?</div>
        <div class="desc-benefits">Los principales beneficios que obtendras al usar nuestra aplicacion son los siguientes</div>
      </div>
      <div class="benefits">
        <div id="w-node-e3a03d2c-d089-fa6c-61db-e9d54e73fe08-fe31b705" class="benefit"><img src="images/Group-25.svg" loading="lazy" alt="" class="image-benefit-1">
          <div class="text-benefit">Seguridad y Protección</div>
        </div>
        <div id="w-node-efa5b219-a3e9-3ae2-fe99-0b0971af4387-fe31b705" class="benefit"><img src="images/Group-24.svg" loading="lazy" alt="" class="image-benefit-2">
          <div class="text-benefit">Acciones rápidas de autoridades</div>
        </div>
        <div class="benefit"><img src="images/Group-42.svg" loading="lazy" alt="" class="image-benefit-3">
          <div class="text-benefit">Privacidad de datos personales</div>
        </div>
        <div class="benefit"><img src="images/Group-22.svg" loading="lazy" alt="" class="image-benefit-4">
          <div class="text-benefit">Comunicación con autoridades</div>
        </div>
      </div>
    </div>
    <div class="about-team"><img src="images/Group-27.svg" loading="lazy" width="431" alt="" class="image-about-team">
      <div class="text-about-team">
        <div class="slogan">Somos un equipo de jóvenes dispuestos a combatir contra la inseguridad ciudadana</div>
        <div class="know-about-team">¿Deseas saber quienes conforman al equipo?</div>
        <a href="#" class="button-about-team w-button">Conócenos</a>
      </div>
    </div>
    <div class="download">
      <div class="text-download">
        <div class="title-download">¿Qué esperas? Empieza a cuidarte mejor ante la delincuencia</div>
        <div class="desc-download">Descarga la aplicación para estar mejor preparados ante cualquier situación que pueda surgir en su día a día</div>
        <div class="desc-download-2">Puedes encontrarlo en:</div><img src="images/Disponible-en-GP.svg" loading="lazy" alt="" class="google-play"><img src="images/Descargalo-en-AS.svg" loading="lazy" alt="" class="app-store">
      </div><img src="images/Group-4.svg" loading="lazy" alt="" class="mockup">
    </div>
  </div>
  <div class="footer">
    <div class="logo-footer"><img src="images/Logo.svg" loading="lazy" alt="" class="logo-footer">
      <div class="copyright">Copyright @2022</div>
    </div>
    <div class="politics-terms">
      <div class="politics">Políticas de privacidad</div>
      <div class="terms">Términos y condiciones</div>
    </div>
    <div class="social"><img src="images/Instagram.svg" loading="lazy" alt="" class="social-1"><img src="images/Facebook.svg" loading="lazy" alt="" class="social-2"><img src="images/Linkedin.svg" loading="lazy" alt="" class="social-3"><img src="images/Twitter.svg" loading="lazy" alt="" class="social-4"></div>
  </div>
  <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=628ed7fe06cabf316a31b704" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
  <script src="js/webflow.js" type="text/javascript"></script>
  
</body>
</html>
